File: TUF/Numbers/FactorOfNumber.java
import java.util.ArrayList;
import java.util.Scanner;

public class FactorOfNumber {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the number: ");
        int num = input.nextInt();
        input.close();
        findFactors(num);
    }

    private static void findFactors(int num) {
        ArrayList<Integer> list = new ArrayList<>();
        for (int i = 1; i <= num; i++) {
            if (num % i == 0) {
                list.add(i);
            }
        }
        // Print the result...
        for (int i = 0; i < list.size(); i++) {
            System.out.print(list.get(i) + " ,");
        }
    }
}

File: TUF/Numbers/FactorOfNumber.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
function findFactors(num) {
    var result = [];
    for (var i = 1; i <= num; i++) {
        if (num % i == 0) {
            result.push(i);
        }
    }
    console.log(result);
}
var num = 6;
findFactors(num);
